README.md

```diff
@@ -175,6 +175,9 @@ $ markdownlint-cli2 "**/*.md" "#node_modules"
 - The format of this file is a [CommonJS module][commonjs-module] that exports
   the object described above for `.markdownlint-cli2.jsonc`.
 - Other details are the same as for `.markdownlint-cli2.jsonc` described above.
+- But, any place that accepts an takes a `string` with the name of
```

Comment:
Please move the new line above the previous about “other details”.

I’m thinking something more like:
> Instead of passing a `String` to identify the module name/path to load for `customRules`, `markdownItPlugins`, and `outputFormatters`, the corresponding  `Object` or `Function` can be provided directly.